Annex60/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -649,8 +649,10 @@ equation
      connect(HDirNor_in, HDirNor_in_internal)
       "Get HDirNor using user input file";
   elseif  HSou == Annex60.BoundaryConditions.Types.RadiationDataSource.Input_HGloHor_HDifHor then
-      (HGloHor_in_internal -HDifHor_in_internal)/Annex60.Utilities.Math.Functions.smoothMax(x1=cos(zenAng.zen), x2=epsCos, deltaX=0.1*epsCos)
-       = HDirNor_in_internal
+      Annex60.Utilities.Math.Functions.smoothMin(
+        x1=((HGloHor_in_internal -HDifHor_in_internal)/Annex60.Utilities.Math.Functions.smoothMax(
+        x1=cos(zenAng.zen), x2=epsCos, deltaX=0.1*epsCos)), x2=solCon, deltaX=1e-2)
```

Comment:
@mwetter in these occasions (deltaX very small compared to solCon) does it still make sense to use `smoothMin`? I.e. can't we use `min()` instead?